cli/lib/kontena/cli/stacks/registry/show_command.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../common'
+
+module Kontena::Cli::Stacks::Registry
+  class ShowCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::Stacks::Common
+    include Kontena::Cli::Stacks::Common::StackNameParam
+
+    banner "Shows information about a stack on the stacks registry"
+
+    option ['-v', '--versions'], :flag, "Only list available versions"
+
+    requires_current_account_token
+
+    def execute
+      stack = ::YAML.load(stacks_client.show(stack_name))
```

Comment:
As this one too.  Using `user/stack:version` showed info from that version and just `user/stack` would show info on the 'latest'.

Will push in the morning from office.